requirements.txt

```diff
@@ -2,5 +2,5 @@ aiohttp==3.4.4
 aiohttp_jinja2==1.1.0
 beautifulsoup4==4.6.3
 cssutils==1.0.2
-gitpython==2.1.11
+gitpython
```

Comment:
Why are you unpinning the version?

Reply:
version 2.1.11 causes run time error in setup.py while using python 3.6 or higher. I unpinned the version to get the latest update of gitpython.

Reply:
fixed in other PR #238 , please rebase you branch

Reply:
@afeena Cool, I was trying to fix the exact same problem as in #238